src/votekit/pref_profile.py

```diff
@@ -1,4 +1,4 @@
-from .ballot import Ballot
+from ballot import Ballot
```

Comment:
most likely you're executing the file directly which is causing it to need this import to be written this way but this will break imports from within the package, since the top level package is named votekit it won't be able to find ballot without the relative path (.)